azure/table_azure_logic_app_workflow.go

```diff
@@ -0,0 +1,276 @@
+package azure
+
+import (
+	"context"
+
+	"github.com/Azure/azure-sdk-for-go/profiles/2020-09-01/monitor/mgmt/insights"
+	"github.com/Azure/azure-sdk-for-go/services/logic/mgmt/2019-05-01/logic"
+	"github.com/turbot/steampipe-plugin-sdk/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/plugin/transform"
+
+	"github.com/turbot/steampipe-plugin-sdk/plugin"
+)
+
+//// TABLE DEFINITION
+
+func tableAzureLogicAppWorkflow(_ context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "azure_logic_app_workflow",
+		Description: "Azure Logic App Workflow",
+		Get: &plugin.GetConfig{
+			KeyColumns:        plugin.AllColumns([]string{"name", "resource_group"}),
+			Hydrate:           getLogicAppWorkflow,
+			ShouldIgnoreError: isNotFoundError([]string{"ResourceNotFound", "ResourceGroupNotFound"}),
```

Comment:
In case of invalid input to `name` we  are getting `400` error code , please check handle this error code.